<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="4" passed="3" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2025-03-14T15:51:29 EDT" name="Default suite" finished-at="2025-03-14T15:51:34 EDT" duration-ms="4125">
    <groups>
    </groups>
    <test started-at="2025-03-14T15:51:29 EDT" name="Default test" finished-at="2025-03-14T15:51:34 EDT" duration-ms="4125">
      <class name="SFS.Smart_Visitor.App">
        <test-method signature="openApp()[pri:1, instance:SFS.Smart_Visitor.App@6b8ca3c8]" started-at="2025-03-14T15:51:29 EDT" name="openApp" finished-at="2025-03-14T15:51:31 EDT" duration-ms="1917" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- openApp -->
        <test-method signature="testLogo()[pri:2, instance:SFS.Smart_Visitor.App@6b8ca3c8]" started-at="2025-03-14T15:51:31 EDT" name="testLogo" finished-at="2025-03-14T15:51:33 EDT" duration-ms="1285" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testLogo -->
        <test-method signature="testlogin()[pri:3, instance:SFS.Smart_Visitor.App@6b8ca3c8]" started-at="2025-03-14T15:51:33 EDT" name="testlogin" finished-at="2025-03-14T15:51:34 EDT" duration-ms="859" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [] but found [https://testclient.smartfoodsafe.com/#/auth/login]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [] but found [https://testclient.smartfoodsafe.com/#/auth/login]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1590)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:656)
at org.testng.Assert.assertEquals(Assert.java:666)
at SFS.Smart_Visitor.App.testlogin(App.java:62)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:687)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:995)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at org.testng.TestRunner.privateRun(TestRunner.java:741)
at org.testng.TestRunner.run(TestRunner.java:616)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testlogin -->
        <test-method signature="testlogout()[pri:4, instance:SFS.Smart_Visitor.App@6b8ca3c8]" started-at="2025-03-14T15:51:34 EDT" name="testlogout" finished-at="2025-03-14T15:51:34 EDT" duration-ms="2" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testlogout -->
      </class> <!-- SFS.Smart_Visitor.App -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
